authorization/src/main/kotlin/com/ritense/authorization/web/PermissionManagementResource.kt

```diff
@@ -21,10 +21,12 @@ import com.ritense.authorization.deployment.PermissionDto
 import com.ritense.authorization.web.request.SearchPermissionsRequest
 import com.ritense.valtimo.contract.domain.ValtimoMediaType
 import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 
+@Controller
```

Comment:
You are right, but the annotation is required since Spring 3, or it will not be used in the requestmapping. 
See my comment on the story: "@Controller or @RestController are now mandatory"

I don't know how we should fix this. Ideally, we should disable the component scanning I think.